pkg/connector/chatsync.go

```diff
@@ -109,7 +109,7 @@ func (tc *TwitterClient) syncChannels(ctx context.Context, inbox *response.Twitt
 						Bool("conv_trusted", conv.Trusted)
 				},
 				PortalKey:    tc.MakePortalKey(conv),
-				CreatePortal: conv.Trusted,
+				CreatePortal: !conv.LowQuality,
```

Comment:
https://github.com/mautrix/twitter/pull/63#issuecomment-2887261909 suggests this is wrong